server/routers/tshirt.py

```diff
@@ -0,0 +1,59 @@
+"""
+T-shirt Registration Route
+"""
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from config.database import get_database
+from config.logger import logger
+from server.models.tshirt import TshirtRequestModel
+from server.models.errors import GenericError
+from server.schemas.tshirt import Tshirt
+from server.schemas.users import Users
+from server.controllers.auth import JWTBearer, decode_jwt
+
+valid_sizes = ["S", "M", "L", "XL", "XXL"]
+
+router = APIRouter(
+    prefix="/register_tshirt",
+)
+
+
+@router.post(
+    "/",
+    dependencies=[Depends(get_database), Depends(JWTBearer())],
+)
+async def register_tshirt(
+    size_requested: TshirtRequestModel,
+    token: str = Depends(JWTBearer()),
+    database: Session = Depends(get_database),
+) -> None:
+    """
+    POST route for Tshirt Registration
+    """
+    try:
+        email = decode_jwt(token)["email"]
+        user_id = database.query(Users.id).filter_by(email=email).first()
+        user_id = 1
```

Comment:
🤓

Reply:
Ah shit, that was for testing 😭 I'll change rn